cell_test.go

```diff
@@ -143,6 +146,9 @@ func (l *CellSuite) TestFormattedValue(c *C) {
 	negativeCell.numFmt = "#,##0 ;[red](#,##0)"
 	c.Assert(negativeCell.FormattedValue(), Equals, "(37947)")
 
+	negativeCell.numFmt = "#,##0.00;(#,##0.00)"
+	c.Assert(negativeCell.FormattedValue(), Equals, "(-37947.75)")
```

Comment:
This matches the actual output of the code. However, based on [this](https://github.com/tealeg/xlsx/blob/master/cell.go#L236), I suspect the code is wrong, and should use `Abs()` to remove the minus sign.

@tealeg